apps/site/assets/ts/schedule/components/schedule-finder/ServiceOptGroup.tsx

```diff
@@ -13,6 +14,42 @@ interface Props {
   multipleWeekdays: boolean;
 }
 
+const optionForService = (
+  service: ServiceByOptGroup,
+  group: ServiceOptGroupName,
+  multipleWeekdays: boolean
+): ReactElement<HTMLElement> => {
+  if (group === "holiday") {
+    return (
+      <HolidayServiceOption
+        key={service.service.id + service.servicePeriod}
+        service={service.service}
+        servicePeriod={service.servicePeriod}
+      />
+    );
+  }
+
+  if (service.service.typicality === "unplanned_disruption") {
```

Comment:
Since the `ServiceOption` itself could make this distinction (as it has the full `service`), would it be better for this conditional to live there and not have a separate `StormServiceOption`?

Reply:
This is basically an artifact of a refactoring that didn't work out, I'll try re-refactoring.